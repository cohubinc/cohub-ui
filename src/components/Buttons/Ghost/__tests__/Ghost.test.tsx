import React from "react";
import { mount } from "enzyme";

import { Buttons } from "dist";

describe("Ghost", () => {
  it("renders without crashing", () => {
    const wrapper = mount(<Buttons.Ghost>push me</Buttons.Ghost>);
    expect(wrapper).toMatchSnapshot();
  });
});
