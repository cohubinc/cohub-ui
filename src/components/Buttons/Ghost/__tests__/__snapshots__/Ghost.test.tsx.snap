// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ghost renders without crashing 1`] = `
<Ghost>
  <Outline
    backgroundColor="hsl(0, 2%, 12%)"
    color="hsl(0, 0%, 98%)"
  >
    <OutlineButtonBase
      backgroundColor="hsl(0, 2%, 12%)"
      color="hsl(0, 0%, 98%)"
    >
      <Base
        backgroundColor="hsl(0, 2%, 12%)"
        raised={false}
        style={
          Object {
            "border": ".75px solid hsl(0, 0%, 98%)",
            "color": "hsl(0, 0%, 98%)",
          }
        }
      >
        <button
          className="CohubButton "
          style={
            Object {
              "backgroundColor": "hsl(0, 2%, 12%)",
              "border": ".75px solid hsl(0, 0%, 98%)",
              "boxShadow": undefined,
              "color": "hsl(0, 0%, 98%)",
            }
          }
        >
          <div
            className="button-text relative flex items-center"
          >
            <Component
              color="hsl(0, 0%, 98%)"
              style={
                Object {
                  "opacity": 1,
                  "transition": "opacity 150ms ease-in",
                }
              }
              uppercase={true}
            >
              <span
                className=""
                data-qa="text"
                style={
                  Object {
                    "color": "hsl(0, 0%, 98%)",
                    "fontFamily": "Inter",
                    "fontSize": 12,
                    "fontWeight": 400,
                    "opacity": 1,
                    "textAlign": "initial",
                    "textTransform": "uppercase",
                    "transition": "opacity 150ms ease-in",
                  }
                }
              >
                <div
                  className="flex items-center"
                  style={
                    Object {
                      "flexDirection": "row",
                    }
                  }
                >
                  <span
                    style={
                      Object {
                        "marginLeft": "",
                        "marginRight": "",
                      }
                    }
                  >
                    push me
                  </span>
                </div>
              </span>
            </Component>
          </div>
        </button>
      </Base>
    </OutlineButtonBase>
  </Outline>
</Ghost>
`;
